orange/src/main/java/com/hartwig/hmftools/orange/conversion/OrangeConversion.java

```diff
@@ -101,8 +102,8 @@ public static LilacAllele convert(@NotNull com.hartwig.hmftools.common.hla.Lilac
     public static VirusInterpreterData convert(@NotNull com.hartwig.hmftools.common.virus.VirusInterpreterData interpreterData)
     {
         return ImmutableVirusInterpreterData.builder()
-                .allViruses(ConversionUtil.mapToIterable(interpreterData.allViruses(), OrangeConversion::convert))
-                .reportableViruses(ConversionUtil.mapToIterable(interpreterData.reportableViruses(), OrangeConversion::convert))
+                .allViruses(ConversionUtil.mapToIterable(VirusInterpreter.filterBlacklistedViruses(interpreterData.allViruses()), OrangeConversion::convert))
+                .reportableViruses(ConversionUtil.mapToIterable(VirusInterpreter.filterBlacklistedViruses(interpreterData.reportableViruses()), OrangeConversion::convert))
```

Comment:
Minor and not 100% sure but you could again make a "VirusInterpreterAlgo" that filters the blacklisted viruses after loading, and before conversion, so we can keep the conversion code trivial (no actual logic)

Reply:
Moved it